src/plugins/janus_streaming.c

```diff
@@ -4178,6 +4189,8 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 		json_t *pin = json_object_get(root, "new_pin");
 		json_t *is_private = json_object_get(root, "new_is_private");
 		json_t *permanent = json_object_get(root, "permanent");
+		json_t *edited_event = json_object_get(root, "edited_event");
+		gboolean send_edited_event = edited_event ? json_is_true(edited_event) : FALSE;
```

Comment:
Since you're adding a new property, it must be added to the object that performs the automated valitation checks on incoming JSON payloads. Look for `static struct janus_json_parameter edit_parameters[]` in the code, where other existing parameters are defined already.

Reply:
This is resolved. I have also added the new_metadata parameter which seems to be also missing.